.gitlab/ci/on-push.yml

```diff
@@ -21,19 +21,6 @@ trigger-private-build:
     - python3 Utils/get_private_build_status.py --github-token $GITHUB_TOKEN
 
 
-run-validations:
-  extends:
-    - .run-validations
-    - .push-rule
```

Comment:
why are we deleting this?

Reply:
I think we are going to leave `validations` and `unittests and lint` in CircleCI for all workflows for now - scheduled or not.